client/app/services/notifications.js

```diff
@@ -11,7 +11,7 @@ if (!Notification) {
 
 const hidden = find(
   ['hidden', 'webkitHidden', 'mozHidden', 'msHidden'],
-  prop => prop in document,
+  prop => ('on' + prop.toLowerCase()) in document,
```

Comment:
I think `hidden` is ok but `visibilitychange` should be prefixed with `on`. No?

Reply:
Oh scratch that - it's used with addEventListener so no need for `on`.
Still, don't see why `onhidden`.

Reply:
Because `document.onhidden`, but `document.addEventListener('hidden', ...)` - `addEventListener` expects event name without `on` prefix

Reply:
But `const hidden` is not used in an event listener.

Reply:
that's also bug :man_facepalming: wrong place